### PR TITLE
Sum additions to siegewar config

### PR DESCRIPTION
**Pull request by NotTolerable**
- Enabled Banner Control Reversal, with a 0.75 multiplier so that its not too op but useful enough where players want to try and get a reversal
- Counter Attacker booster, basically gives 5 extra points per kill PER enemy player who capped banner
- Now, blocks can't be destroyed within a 2 block radius of banner (on the same block layer as banner) so that ppl have footing to cap banner
- Siege Immunity timer from 9 days ->7 days
- Increased balance cap from 15000 -> 20000
(Check out artefacts features : you can make kb 2 sword or like sharpness 6 sword a reward to top nations basically like the superweapons that used to be on ceedric.com)

### CODE REVIEW

File: plugins/SiegeWar/config.yml [hunk 1]
```diff
@@ -678,18 +678,18 @@ trap_warfare_mitigation:
   # TIP: 
   # If this feature is enabled, 
   # then it is recommended to also have a server rule requiring attackers to remove all traps in the area before the banner is placed
-  enabled: 'false'
+  enabled: 'true'
```
Comment (ceeedric) [inline plugins/SiegeWar/config.yml]: Keep this disabled. Equinox had this feature and it just felt stupid that you couldn't mine around the banner even when they lowered the radius

File: plugins/SiegeWar/config.yml [hunk 2]
```diff
@@ -319,12 +319,12 @@ war:
         # If this setting is true, then if a player from the banner controlling side dies,
         # the death points are increased by a certain percentage.
         # TIP: This feature delivers extra balance to battles in which one side is outnumbered by the other.
-        enabled: 'false'
+        enabled: 'true'
```
Comment (ceeedric) [inline plugins/SiegeWar/config.yml]: This would increase capping warfare and alt capping. Sure it is a low amount but I'd rather keep it disabled

**Review by ceeedric — CHANGES_REQUESTED** [review]
I prefer not to have counter attack booster and banner radius protection

Anyone cloning this repository can change this

**Comment by NotTolerable** [comment]
ok

**Comment by NotTolerable** [comment]
Ok will do

**Review by ceeedric — APPROVED** [review]

**Pull request merged**